tests/search/registry.json

```diff
@@ -0,0 +1,17 @@
+{
+    "flakes": [
+        {
+            "from": {
+                "id": "whatever",
```

Comment:
question: is this file being used? I don't see any instances of `whatever`

Reply:
No, we're kicking the can down the road. I can put that on the to-do list as well in the next PR.

Reply:
For context, the next PR is a direct follow up to this one that properly formats the search output.

Reply:
Fixed